{{ cookiecutter.safe_formal_name }}/app/src/main/python-briefcase/rubicon/java/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Emulation layer to make Chaquopy look like Rubicon."""
+
+import java
+
+JavaClass = java.jclass
+
+def JavaInterface(name):
+    # FIXME won't work if a class implements multiple interfaces: does Toga need that?
+    return java.dynamic_proxy(java.jclass(name))
```

Comment:
You're correct that Rubicon doesn't currently support multiple interfaces on a class; we're only ever implementing a callback interface for a widget, so complex class hierarchies haven't ever been needed.

I wouldn't be *overly* concerned about code in the wild interacting with Android APIs directly; Rubicon-Java is sufficiently in the early-stage (and undocumented) that I'm comfortable with relatively major backwards incompatibilities. That said, if the problem is relatively easy to avoid, then that's obviously preferable.